vault/dendron.guides.workflows.bullet-journal.md

```diff
@@ -17,7 +17,7 @@ Bullet journalling is a way of quickly getting your thoughts down for the day. I
 
 ### Create Daily Journal Notes Command
 
-![[dendron.topic.special-notes#daily-journal,1:#*]]
+![[dendron.topic.special-notes#journal-note,1:#*]]
```

Comment:
```suggestion
![[dendron://dendron.dendron-site/dendron.topic.daily-journal-note#details,1]]
```

Reply:
Daily journal had moved to being a standalone note, so I've updated your update :)